src/utils/registryFileUtils.js

```diff
@@ -23,6 +23,9 @@
  *      "bar": { "name": "bar", "url": "...", "token": "..." }
  *    }
  * }
+ *
+ * TODO : Defaul name for refocus collector is Refocus-Collector
+ * which can be modified by providing name.
```

Comment:
It can be modified by providing when executing start command? If yes, can you add that in comment here please?

Reply:
No When you create collector. Yes I will update comment.